viewer/packages/tools/src/Measurement/MeasurementTool.ts

```diff
@@ -248,6 +250,7 @@ export class MeasurementTool extends Cognite3DViewerToolBase {
     this._viewer.off('click', this._handlePointerClick);
     this._events.measurementEnded.fire();
     this._measurementMode = false;
+    this._showMeasurements = true;
```

Comment:
Does it make sense to always show measurements when exiting the measurement mode?